docs/apache-airflow/core-concepts/dags.rst

```diff
@@ -712,19 +712,10 @@ configuration parameter (*added in Airflow 2.3*): ``regexp`` and ``glob``.
 
 .. note::
 
-    The default ``DAG_IGNORE_FILE_SYNTAX`` is ``regexp`` to ensure backwards compatibility.
+    The default ``DAG_IGNORE_FILE_SYNTAX`` is ``glob`` in Airflow 3 or later.
+    The default was ``regexp`` in previous Airflow versions.
```

Comment:
```suggestion
    The default ``DAG_IGNORE_FILE_SYNTAX`` is ``glob`` in Airflow 3 or later (in previous versions it was ``regexp``).
```

Reply:
The two basically read the same to me (in meaning)? Happy to change if you prefer this, but not sure if I’m missing something.

Reply:
Just to make this sentence shorter and in parathensis, as it discusses a previous version :) I prefer this, but not too critical.